sdk/python/setup.py

```diff
@@ -40,7 +40,7 @@
 REQUIRES_PYTHON = ">=3.7.0"
 
 REQUIRED = [
-    "Click==8.*",
+    "Click>=7.*",
```

Comment:
Was this necessary for the same reason (python 3.7 support)?

Reply:
ah no this was just a fix i wanted to sneak in from a previous PR because it seemed overly restrictive

Reply:
sg. stamping.

Reply:
@adchia @achals I think this was a typo as pip actually interprets this as `click==7.*`. See https://github.com/feast-dev/feast/pull/2450